src/components/MagicSyncViewer/components/MagicSyncViewerStactionbar.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import * as styles from './MagicSyncViewerStactionbar.scss';
+import IReactComponentProps from '../../../common/structures/IReactComponentProps';
+import FlySelect from '../../FlySelect';
+import { ConnectDirectionType } from '../helpers/magicSyncViewMock';
+
+interface IMagicSyncViewerHeaderProps extends IReactComponentProps {
+	connectDirection: ConnectDirectionType;
+}
+
+export const MagicSyncViewerStactionbar = (props: IMagicSyncViewerHeaderProps) => (
```

Comment:
Naming is a bit confusing here. Might be worth sticking to one half of the portmanteau 😃

Reply:
Ha, absolutely! I'll change it to `statusbar`. Good call.